lib/oauthUtil.js

```diff
@@ -0,0 +1,188 @@
+/* eslint-disable complexity, max-statements */
+var http = require('./http');
+var util = require('./util');
+var AuthSdkError = require('./errors/AuthSdkError');
+
+function isToken(obj) {
+  if (obj &&
+      (obj.accessToken || obj.idToken) &&
+      Array.isArray(obj.scopes)) {
+    return true;
+  }
+  return false;
+}
+
+function addListener(eventTarget, name, fn) {
+  if (eventTarget.addEventListener) {
```

Comment:
I wonder if we should remove this logic since addEventListener is supported in IE9+